src/AppInstallerCommonCore/Manifest/ManifestValidation.cpp

```diff
@@ -398,4 +447,9 @@ namespace AppInstaller::Manifest
 
         return errors;
     }
+
+    std::string_view ValidationError::GetErrorMessage() const
+    {
+        return ErrorIdToMessageMap[Message];

```

Comment:
Maybe if the StringID is not in the map it would be better to return the ID instead of the empty string.